packages/polar/src/lib/deploy/contract.ts

```diff
@@ -109,8 +108,9 @@ export class Contract {
     [name: string]: ContractFunction<any> // eslint-disable-line  @typescript-eslint/no-explicit-any
   };
 
-  constructor (contractName: string) {
+  constructor (contractName: string, customFees?: Partial<FeeTable>) {
```

Comment:
customFee should be fetched from config file and type checking should be done. type FeeTable will be used to typecheck inputs